.circleci/config.yml

```diff
@@ -68,4 +68,15 @@ jobs:
       - add_ssh_keys:
           fingerprints:
             - "79:26:19:57:4e:1a:59:64:ec:bb:b7:76:e2:5f:7c:ad"
-      - run: yarn deploy
+      - run:
+          name: Compile and deploy Github pages
+          command: |
+            yarn build
+            git checkout gh-pages
+            rm -rf $CIRCLE_BRANCH
+            mv build $CIRCLE_BRANCH
+            git add .
+            git diff-index --quiet HEAD || git commit -m 'Update Github pages for $CIRCLE_BRANCH'
+            git push origin gh-pages
```

Comment:
I'd like to see our main site preserved with a branch we deem "stable"

![image](https://user-images.githubusercontent.com/142403/57105294-443f0b80-6cdf-11e9-9e1f-d9a83ca2f4a8.png)

Reply:
Ah, that is now in the master folder.

Reply:
https://mavenlink.github.io/design-system/master/

Does that work for you?

Reply:
Yes, so I saw that, but I think maybe we just need to figure out how to alias the root site to `master` too. Is that going to prove difficult?

My reasoning:
* It's a very common thing for folks to manually go to top level of a site and that should not cause 404
* We link to it from our READMEs currently (obviously fixable, but I still think top level should work imo)

Reply:
I think I am going to place an `index.html` with a link to the `master/` then. I would prefer to start the new process of always specifying the branch instead of an alias or redirect. We can gather some feedback later and make corrective actions then.

Reply:
Done @ https://mavenlink.github.io/design-system/

Reply:
why not just do:
`window.location.replace("https://mavenlink.github.io/design-system/master/");`